docs/fides/docs/installation/docker.md

```diff
@@ -1,9 +1,36 @@
 
-The fidesctl community releases Docker Images which are reference images for fidesctl. Every time a new version of fidesctl is released, the images are available in the [ethyca/fidesctl DockerHub](https://hub.docker.com/r/ethyca/fidesctl/tags). There are also mid-release versions (dirty versions) that get uploaded to DockerHub on every commit to the main branch.
+The fidesctl community releases a production-ready reference container image of fidesctl for the ease of deployment in production.
 
-These reference images contain all of the extras and dependencies for running the Python application. However they do not contain the required Postgres database.
+## Docker Hub Releases
 
-Fidesctl requires multiple components to function as it is a multi-part application. You may therefore also be interested in launching fidesctl in the Docker Compose environment, see: [Running Fidesctl in Docker](../quickstart/docker.md).
+The following is an overview of the fidesctl Docker Hub release process, including release cadence and what's included. 
+
+**Release Cadences**
+
+Every time a new version of fidesctl is released, the images are available in the [ethyca/fidesctl DockerHub](https://hub.docker.com/r/ethyca/fidesctl/tags).
+
+There are also mid-release versions (dirty versions) that get uploaded to DockerHub on every commit to the main branch. Implementation is not finalized in mid-release versions. 
+
+**What's Included**
+
+These reference images contain all of the extras and dependencies for running the Python application. However they do not contain the required Postgres database. Continue to [Setting up the database](./database.md).
+
+## Installation
+
+Install Docker and Docker Compose, then pull the latest container image from Docker Hub.
+
+1. Install `docker` locally (see [Docker Desktop](https://www.docker.com/products/docker-desktop) or your preferred installation). The minimum verified Docker version is `20.10.8`
+2. If your `docker` installation did not include `docker-compose`, make sure to get at least version `1.29.0`. Installation instructions can be found in the [Docker Compose documentation](https://docs.docker.com/compose/install/).
+3. Run `docker pull ethyca/fidesctl` to pull the latest fidesctl container image from Docker Hub.
+
+Congratulations! Fidesctl is now installed on your machine. 
+## Next Steps
+
+You're now ready to start enforcing privacy with Fidesctl! Below are suggestions for continuing your journey.
+
+- See  [Setting up the database](./database.md) to set up the Postgresql for fidesctl.
+- See [Running Fidesctl in Docker](../quickstart/docker.md) for more information on launching fidesctl in Docker Compose. Fidesctl requires multiple components to function as it is a multi-part application.
```

Comment:
"Running Fidesctl in Docker" contains the same instructions you've included here from the System Requirements section. In your opinion, does it make sense to have it in both places? (this is not a leading question! and you don't have to update the PR if you don't feel it's necessary, I want to understand your opinion on this)
![Screen Shot 2022-01-26 at 8 33 33 AM](https://user-images.githubusercontent.com/85575406/151172333-9ed3569f-fb75-4e8b-879d-298d84279857.png)

Reply:
This is a great question! 

In my last role, we had reusable files that would be able to be shared across page for shared instructions like this. 

I think a solution like that, or simply linking to System Requirements would be best here. Having the same instructions in too places will just lead to tech debt in the future. 

For now, I'll update with a link to System Requirements.